### DISCUSSION TITLE
Can I change location of the logo file?

### DISCUSSION THREAD

**Discussion by ppiegaze** [vuejs/vitepress#3078] [Q&A]
In my `config.mts` I have the following:

```
export default defineConfig({
  ...
  srcDir: "./src",
  themeConfig: {
    logo: "/logo.svg",
    ...
    }
}
```

As you can see I have used the `srcDir` property to move the root of my MD content to `<project-root>/src`. I did this because I want to isolate the site directory structure and not mix them in with the other files and dirs in the `<project-root>`.

Unfortunately, in order for vitepress to find the `logo.svg` file, it apparently has to be here:
`<project-root>/src/public/logo.svg`

I have tried moving the file to `<project-root>/images/logo.svg` (where the rest of my images reside) and adjusting the `config.mts` to 
```
...
themeConfig: {
    logo: "../images/logo.svg",
    ...
    }
...
```

But vitepress does not seem to find the file.

Is there some way to set the `logo` property so that I can place my `logo.svg` file outside `<project-root>/src/public/` (and, indeed, outside `<project-rrot>/src/` entirely)?

**Comment by brc-dd** [comment]
Unfortunately there is no direct way. Although there might be some alternatives like copy that file to output directory after build, or maybe instead of configuring srcDir, configure rewrites, then you can have root/public/images 👀
